test/common/wsServerMaster.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var ChildProcess = require('child_process');
+var path = require('path');
```

Comment:
WS server running on separate process covers functionality of the same process wsServer - I think the logic should be reused and wsServer file removed

Reply:
The dependent tests are complex and have redundant use of `wsServer`. Already spent more than hour with you to pass ws-functional tests with new process. I used `wsServerMaster` only where appropriate right now. Refactoring all existing usage, may be out of scope for this PR.

Reply:
Agree. Another PR can do that refactoring.